Configuration/TCA/Overrides/tt_content.php

```diff
@@ -27,7 +27,7 @@
     foreach ($pluginNames as $pluginName => $pluginConfig) {
         $pluginSignature = 'cart_' . strtolower($pluginName);
 
-        ExtensionUtility::registerPlugin(
+        $contentTypeName = ExtensionUtility::registerPlugin(
```

Comment:
The variable `$contentTypeName` is not required, as it should match `$pluginSignature`.

```suggestion
        ExtensionUtility::registerPlugin(
```